CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
```

Comment:
What do you think about following this convention?  https://keepachangelog.com/en/1.0.0/

Reply:
I'm fine either way. I based this one on the Capistrano standard, mostly because it's what I'm used to.

Reply:
Updated to follow that convention, mostly.